src/main/java/seedu/duke/atomichabit/command/AddCommand.java

```diff
@@ -0,0 +1,30 @@
+package seedu.duke.atomichabit.command;
+
+import seedu.duke.atomichabit.feature.AtomicHabit;
+import seedu.duke.atomichabit.feature.AtomicHabitList;
+
+public class AddCommand extends Command {
+    private String description;
+    private static final String FIRST_STRING = "Yay! You have added a new habit:";
+    private static final String SECOND_STRING = " was successfully added";
```

Comment:
For consideration: Is `FIRST_STRING` and `SECOND_STRING` descriptive? What if addcommand needs to print other messages?